tasks/gulp/build.js

```diff
@@ -1,12 +1,16 @@
 const { task, src, dest } = require('gulp')
-const debug = require('gulp-debug')
```

Comment:
If we're removing this we can also remove `gulp-debug` from `package.json`? Might want to do as a separate commit.

Reply:
Good spot, I've removed gulp-debug